SU-SWS/stanford_basic
# D8CORE-938: Menu tweaks.

# READY FOR REVIEW

# Summary
- Updates Decanter to 6.0.1
- Updates Webpack settings
- Updates Secondary and Multi menu components in Stanford Basic both visually and accessibility wise (need to abstract out to Decanter - later)
- Refactor of Twig templates to remove the dependency on ui_patterns twig function `pattern()`
- Changed Twig Namespaces to give `dist/templates` (`@basic-dist`) it's own name instead of being opaquely wrapped into the `@basic` namespace.
- Fixes bug where all status messages were getting the same type.

# Review By (Date)
- EOD Friday if possible

# Urgency
- Med/High - Would like to roll out into a new release sooner than later and as soon as this work is finalized I can abstract out and into Decanter.

# Steps to Test

1. Check out this branch on a working site or visit either the tugboat build in https://github.com/SU-SWS/acsf-cardinalsites/pull/46 or test build at https://navigation-test.sites.stanford.edu/
2. Review aesthetics on mobile and desktop
3. Review voiceover and keyboard controls on mobile and desktop

# Affected Projects or Products
- Pretty much all D8Core & SOE D8 Development

# Associated Issues and/or People
- D8CORE-901 and linked tasks
- @yvonnetangsu FYI
- @kerri-augenstein FYI
- @katrialesser FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
